### DISCUSSION TITLE
RFUS y291ga 9.0.35 Allwinner-V2 + gdrive?

### DISCUSSION THREAD

**Discussion by borowa1212** [roleoroleo/yi-hack-Allwinner-v2#658] [Q&A]
In the beginning, I would like to say sorry, for every lame question ;D

I've read all the treads related to my camera - as in subject, but found nothing about gdrive

I have two YI home 1080p cameras (RFUSY620BB - 9.0.35 - so y291ga)
Would the Allwinner-v2 will work with gdrive? I really want the camera to send the movies to the cloud, because I have it in my hotel room, therefore lot of people have an access to the SD card.

If Allwinner-v2 is not working with gdrive repo, is it possible to use any other repo which will work with my camera and gdrive?

Carefull right now, lame question ;D - where can I find gdrive repo? Should I just download the code from here `https://github.com/roleoroleo/yi-hack-Allwinner.gdrive` as a *.zip, unpack repository, and copy all the folders to yi-hack folder to SD?

I'm rather a guy which want to ask, to prevent some unnecessary misunderstandings, and mistakes.

May I, flash both repos at the same time -> allwinner-v2 + gdrive? I'm planning to put the gdrive folders to yi-hack in which I put allwinner-v2 and place the SD in the camera.

I will be really grateful for your help and understanding ;D

**Comment by roleoroleo** [comment]
> I have two YI home 1080p cameras (RFUSY620BB - 9.0.35 - so y291ga) Would the Allwinner-v2 will work with gdrive? I really want the camera to send the movies to the cloud, because I have it in my hotel room, therefore lot of people have an access to the SD card.

I published this repo:
https://github.com/roleoroleo/yi-hack-Allwinner.gdrive
as a custom copy of:
https://github.com/porunov/xiaomi_gdrive

But I don't update it very often because I don't use this feature and I don't know if it's working now.

> If Allwinner-v2 is not working with gdrive repo, is it possible to use any other repo which will work with my camera and gdrive?

I don't know other repos with a similar feature.

> Carefull right now, lame question ;D - where can I find gdrive repo? Should I just download the code from here `https://github.com/roleoroleo/yi-hack-Allwinner.gdrive` as a *.zip, unpack repository, and copy all the folders to yi-hack folder to SD?

1st step:
you have to install my hack for your model.
2nd step:
follow the instructions here https://github.com/roleoroleo/yi-hack-Allwinner.gdrive#step-by-step-instruction-for-installing-yi_gdrive

> May I, flash both repos at the same time -> allwinner-v2 + gdrive? I'm planning to put the gdrive folders to yi-hack in which I put allwinner-v2 and place the SD in the camera.

It's not a real "flash", it's an addon firmware.
You need to copy the hack in the sd. The cam will apply a little hack but the main processes will run from the sd card.
So, if you remove the sd card the cam will restart with factory features.
About gdrive feature, simply follow the instructions above. Same behavior: if you remove the sd card the cam will restart with factory features.
